### PR TITLE
Update rosa describe machinepool command with --machinepool flag

### PR DESCRIPTION
**Pull request by aaraj7**
Relates to card [OCM-7537](https://issues.redhat.com//browse/OCM-7537)

### CODE REVIEW

File: ci-operator/step-registry/rosa/cluster/wait-ready/nodes/rosa-cluster-wait-ready-nodes-commands.sh [hunk 1]
```diff
@@ -179,9 +179,9 @@ function getDesiredComputeCount {
       if [[ "$compute_count" = "null" ]]; then 
         echo "--auto-scaling enabled, retrieving min_replica count desired"
         if [[ $HOSTED_CP = "true" ]];then
-          compute_count=$(rosa describe machinepool  ${MP_NAME} -c "$CLUSTER_ID" -o json  | jq -r '.autoscaling.min_replica') 
+          compute_count=$(rosa describe machinepool --machinepool ${MP_NAME} -c "$CLUSTER_ID" -o json  | jq -r '.autoscaling.min_replica')
```
Comment (radtriste) [inline ci-operator/step-registry/rosa/cluster/wait-ready/nodes/rosa-cluster-wait-ready-nodes-commands.sh:182]: Please update line #178 also

Comment (aaraj7) [reply ci-operator/step-registry/rosa/cluster/wait-ready/nodes/rosa-cluster-wait-ready-nodes-commands.sh:182]: Done Thanks for that

**Review by radtriste — APPROVED** [review]
/lgtm

**Pull request merged**